lib/plugins/output-filter/lowercase-fields.js

```diff
@@ -0,0 +1,43 @@
+const get = require('get-value')
+const unset = require('unset-value')
+const set = require('set-value')
+
+function renameFields (context, config, eventEmitter, data, callback) {
+  if (data === undefined) {
+    return callback(new Error('data is null'), null)
+  }
+  if (config.allFields !== true) {
+    config.allFields = false
+  }
+
+  try {
+    if (!config.matchSource.test(context.sourceName || data.logSource)) {
+      return callback(null, data)
+    }
+
+    if (config.allFields === true) {
+      const fields = Object.keys(data)
```

Comment:
I take it that this only works for root keys? Does it work for something like `myobject.WHY_ARE_YOU_YELLING_THE_FIELD_NAME` ?

Reply:
Yes, only root keys for now. Let me think about if I can edit it to cascade the lowercasing across all keys as well.